package/src/components/Message/Message.tsx

```diff
@@ -730,6 +753,12 @@ const MessageWithContext = <
         >
           <MessageProvider value={messageContext}>
             <MessageSimple />
+            {isBounceDialogOpen && (
+              <MessageBounce
+                isBounceDialogOpen={isBounceDialogOpen}
```

Comment:
```suggestion
```

this prop is very unnecessary, if its not there we dont render, then why pass it?